docs/architecture/tls13-experimental.md

```diff
@@ -66,3 +66,245 @@ together with their level of testing:
   as part of `MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL`:
 
   - Reader ([`library/mps_reader.h`](../../library/mps_reader.h))
+
+
+MVP definition
+--------------
+
+The TLS 1.3 MVP implements only the client side of the protocol.
+The TLS 1.3 MVP does not support the handling of server HelloRetryRequest and
+CertificateRequest messages. If it receives one of those messages, it aborts
```

Comment:
I was thinking: this makes sense to people who are familiar with the protocol, but for the benefit of the majority who isn't, it's probably better to say something like "The TLS 1.3 MVP does not support client-side authentication. If the server requests it, ...". Or perhaps even start by give some context: "In TLS, the server is always authenticated, but authentication of the client is optional." before describing what the MVP does (not) support.

Reply:
Speaking of people who aren't that familiar with the protocol, I'd welcome a (pointer to an) explanation of the impact of not supporting HelloRetryRequest in practice.

Reply:
I've added an "Overview" section.